platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -117,16 +135,19 @@ private void solderMinimumGenerationNonAncient() {
      * Wire the components together.
      */
     private void wire() {
-        internalEventValidatorWiring.eventOutput().solderTo(eventDeduplicatorWiring.eventInput());
-        eventDeduplicatorWiring.eventOutput().solderTo(eventSignatureValidatorWiring.eventInput());
-        eventSignatureValidatorWiring.eventOutput().solderTo(orphanBufferWiring.eventInput());
-        orphanBufferWiring.eventOutput().solderTo(inOrderLinkerWiring.eventInput());
-        orphanBufferWiring.eventOutput().solderTo(eventCreationManagerWiring.eventInput());
-        inOrderLinkerWiring.eventOutput().solderTo(linkedEventIntakeWiring.eventInput());
-
-        eventCreationManagerWiring.newEventOutput().solderTo(internalEventValidatorWiring.eventInput(), INJECT);
+        if (!platformContext.getConfiguration().getConfigData(EventConfig.class).useLegacyIntake()) {
+            internalEventValidatorWiring.eventOutput().solderTo(eventDeduplicatorWiring.eventInput());
+            eventDeduplicatorWiring.eventOutput().solderTo(eventSignatureValidatorWiring.eventInput());
+            eventSignatureValidatorWiring.eventOutput().solderTo(orphanBufferWiring.eventInput());
+            orphanBufferWiring.eventOutput().solderTo(inOrderLinkerWiring.eventInput());
+            inOrderLinkerWiring.eventOutput().solderTo(linkedEventIntakeWiring.eventInput());
+            orphanBufferWiring.eventOutput().solderTo(eventCreationManagerWiring.eventInput());
+            eventCreationManagerWiring.newEventOutput().solderTo(internalEventValidatorWiring.eventInput(), INJECT);
+
+            solderMinimumGenerationNonAncient();
+        }
 
-        solderMinimumGenerationNonAncient();
+        // TODO we need to wire in the event creator!!
```

Comment:
Has this been done?

Reply:
DOH! Yeah, I wired it. Just forgot to remove my reminder. It's cleaned up now.